gtsfm/bundle/bundle_adjustment.py

```diff
@@ -167,14 +170,25 @@ def get_metrics_from_sfm_data(sfm_data: GtsfmData, suffix: str) -> List[GtsfmMet
         # filter the largest errors
         filtered_result = optimized_data.filter_landmarks(self.output_reproj_error_thresh)
 
-        if wTi_list_gt is not None:
+        if cameras_gt is not None:
+            poses_gt = [cam.pose() for cam in cameras_gt]
+
             # align the sparse multi-view estimate after BA to the ground truth pose graph.
-            filtered_result = filtered_result.align_via_Sim3_to_poses(wTi_list_gt)
+            filtered_result = filtered_result.align_via_Sim3_to_poses(wTi_list_ref=poses_gt)
             ba_pose_error_metrics = metrics_utils.compute_ba_pose_metrics(
-                gt_wTi_list=wTi_list_gt, ba_output=filtered_result
+                gt_wTi_list=poses_gt, ba_output=filtered_result
             )
             ba_metrics.extend(metrics_group=ba_pose_error_metrics)
 
+            output_tracks_exit_codes = track_utils.classify_tracks3d_with_gt_cameras(
+                tracks=filtered_result.get_tracks(), cameras_gt=cameras_gt
+            )
+            output_tracks_exit_codes_distribution = Counter(output_tracks_exit_codes)
+
+            for exit_code, count in output_tracks_exit_codes_distribution.items():
```

Comment:
@ayushbaid maybe we can add an inline comment explaining what this metric is for/ what insights we can take from it?

Reply:
I think this is too much verbosity in the code comments. I can maybe write that we are computing what the tracks "look like", with the GT cameras. 

Suggesting interpretations of the metrics should either go in the report, or we should rename the metrics to be self-explanatory.

What do you think?